.github/workflows/pre-release.yml

```diff
@@ -35,8 +35,11 @@ jobs:
             org.opencontainers.image.created=${{ env.COMMITED_AT }}
             org.opencontainers.image.version=v${{ env.VERSION }}
             org.opencontainers.image.maintainer=GeoWerkstatt GmbH <support@geowerkstatt.ch>
+          flavor: |
+            latest=false
           tags: |
             type=edge
+            type=semver,pattern=v{{version}},value=${{ env.VERSION }}
```

Comment:
@flenny Neu wird pro Pre-Release die Version ebenfalls als Tag gesetzt. Ist für mich verständlicher, siehst du das auch so?

Reply:
Das geht für mich in Orndung. Die aktuelle Lösung sorgt allerdings für ein übersichtliches Image Repo [hier](https://github.com/GeoWerkstatt/interlis-check-service/pkgs/container/interlis-check-service). Da wir nicht veröffentlichte (released) Versionen nirgends deployen würde ich sie eher weglassen. Geht aber wie gesagt in Ordnung.